src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionKind.cs

```diff
@@ -8,6 +8,7 @@ namespace Microsoft.CodeAnalysis.CSharp
 {
     internal enum ConversionKind : byte
     {
+        UnsetConversionKind = 0,
```

Comment:
Why make this the 0 value here vs. adding it to the end of the `enum`?

Reply:
Because it represents the value of a ConversionKind field that hasn't yet been set.  It must be 0 for that to work.

---
In reply to: [260403775](https://github.com/dotnet/roslyn/pull/33648#discussion_r260403775) [](ancestors = 260403775)